python/lsst/obs/hsc/hscMapper.py

```diff
@@ -181,11 +182,19 @@ def std_raw_md(self, md, dataId):
 
         return md
 
-    def std_raw(self, item, dataId):
-        exp = super(HscMapper, self).std_raw(item, dataId)
-        exp, wcs = self._flipChipsLR(exp, exp.getWcs(), dataId)
-        exp.setWcs(wcs)
-        return exp
+    def _createSkyWcsFromMetadata(self, exposure):
```

Comment:
Please add one-liner docstring (since nothing is returned and it's pretty self-explanatory)

Reply:
Docstring comes from parent class.